Competition2017/src/competition/subsystems/shooter_wheel/commands/ContinueShooterCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.shooter_wheel.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ContinueShooterCommand extends BaseCommand {
+
+    ShooterWheelSubsystem shooterWheel;
+
+    @Inject
+    public ContinueShooterCommand() {
+    }
+    
+    public void setSide(ShooterWheelSubsystem shooterWheel) {
+        this.shooterWheel = shooterWheel;
+        this.requires(shooterWheel);
+    }
+
+    @Override
+    public void initialize() {
+    
```

Comment:
There's another task to refactor these commands completely, I don't think we need to do anything to them about how they currently work